### PR TITLE
WireServerEndpoint changes

### PR DESCRIPTION
**Pull request by Jyotsna-Anand**

### CODE REVIEW

File: VMEncryption/main/EncryptionSettingsUtil.py [hunk 1]
```diff
@@ -227,14 +227,33 @@ def get_http_util(self):
         from HttpUtil import HttpUtil
         return HttpUtil(self.logger)
 
+	def get_wireserver_endpoint_uri(self):
+
+		wireserver_endpoint_file = '/var/lib/waagent/WireServerEndpoint'
```
Comment (ejarvi) [inline VMEncryption/main/EncryptionSettingsUtil.py:232]: would be cleaner to define this in Common.py alongside the wire server IP constant

File: VMEncryption/main/Common.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ class CommonVariables:
     encryption_settings_counter_file = 'counter.txt'
     encryption_settings_counter_path = encryption_key_mount_point + '/' + encryption_settings_counter_file
 
-    wireserver_endpoint = "http://169.254.169.254:80/machine?comp=diskEncryptionData"
+	wireserverIP = "168.63.129.16"
```
Comment (ejarvi) [inline VMEncryption/main/Common.py:50]: maybe call this static_wireserver_ip to differentiate it from the dynamic one

File: VMEncryption/main/EncryptionSettingsUtil.py [hunk 3]
```diff
@@ -227,14 +227,33 @@ def get_http_util(self):
...
+		wireserverIP = None
+
+		if os.path.exists(wireserver_endpoint_file):
+		    with open(wireserver_endpoint_file, 'r') as wip:
+			    wireserverIP = wip.read()
+			    self.logger.log("wireserverIP found in {0} = {1}".format(wireserver_endpoint_file, wireserverIP))
+
+		if wireserverIP == None:
+            self.logger.log("Using static wireServerIP from CommonVariables")
```
Comment (ejarvi) [inline VMEncryption/main/EncryptionSettingsUtil.py:241]: indentation

**Review by ejarvi — CHANGES_REQUESTED** [review]

File: VMEncryption/main/EncryptionSettingsUtil.py [hunk 4]
```diff
@@ -227,14 +227,33 @@ def get_http_util(self):
...
+		    wireserverIP = CommonVariables.wireserverIP
+
+		wireserver_endpoint = "http://" + wireserverIP + ":80/machine?comp=diskEncryptionData"
```
Comment (ejarvi) [inline VMEncryption/main/EncryptionSettingsUtil.py:244]: maybe this format string (that ends in ":80/machine?comp=diskEncryptionData") can be a constant in Common.py as well

File: VMEncryption/main/Common.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jyotsna-Anand) [reply VMEncryption/main/Common.py]: Addressed review comments in commit 21ba15.

**Comment by Jyotsna-Anand** [comment]
This PR was created for an initial review until access to azure-linux-extensions repo could be got. 
Moving this PR to a new branch under "azure-linux-extensions" repo.
https://github.com/Azure/azure-linux-extensions/pull/1209

**Pull request closed without merging**